### DISCUSSION TITLE
Swell Count in Takio mode does not use the classic skin.

### DISCUSSION THREAD

**Discussion by Jan-Arbus** [ppy/osu#18781] [Q&A] [answered]
![屏幕截图 2022-06-20 205937](https://user-images.githubusercontent.com/74862110/174607209-caa67699-19a3-42e8-920e-c29e1fde8c2c.png)
I wonder if this is intentional?

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/issues/13601

please search
